src/main/java/school/lemon/changerequest/java/introduction/hw1/Task7.java

```diff
@@ -3,6 +3,13 @@
 
 public class Task7 {
     public static String getDayTitle(int dayIndex) {
-        return "";
-    }
-}
+        switch(dayIndex){
+            case 1: return "Monday";
+            case 2: return  "Tuesday";
+            case 3: return  "Wednesday";
+            case 4: return  "Thursday";
+            case 5: return  "Friday";
+            case 6: return  "Saturday";
+            case 7: return "Sunday";}
+        return "error";
```

Comment:
use `default` case instead of simple return statement to show that it's also valid and is handled with all other cases.